app/src/main/java/com/teamltt/carcare/DemoActivity.java

```diff
@@ -75,6 +154,9 @@ protected void onCreate(Bundle savedInstanceState) {
         IntentFilter filter = new IntentFilter(BluetoothDevice.ACTION_FOUND);
         filter.addAction(BluetoothAdapter.ACTION_DISCOVERY_FINISHED);
         registerReceiver(bluetoothReceiver, filter);
+
+        // Will this always be the UUID in our case?
+        uuid = UUID.fromString("00001101-0000-1000-8000-00805F9B34FB");
```

Comment:
I found the [android documentation](https://developer.android.com/reference/android/bluetooth/BluetoothDevice.html) for this:

Relevant quote "Hint: If you are connecting to a Bluetooth serial board then try using the well-known SPP UUID 00001101-0000-1000-8000-00805F9B34FB"
I will move this up to a member. When the UI is more stable, we can give the user an  advanced option to set the UUID in case there is a smarter OBDII bluetooth adapter we don't know about that doesn't use the serial port